### ISSUE TITLE
Projects near me failing in Liège

### ISSUE THREAD

**Issue by madewild** [ec-doris/kohesio.eu#502]
Projects near me does not alway display Belgium's projects. The only pattern we could find with Pascal is that laptop within VPN network area will display Belgian projects but personnal computers from external zone will not display. Tested with IPAD, Personal desktop, PGA's Unisys laptop and personal IPhone. Cached have been refreshed and when test failed, tried with "private" session.

**Comment by madewild** [comment]
For me works perfectly with VOO (personal laptop) in Mons:
![pnm](https://user-images.githubusercontent.com/942488/154575759-26bd50ab-abb4-42d0-a9d7-bd58a054326e.png)

Happy to debug with you @djang0 @gachpas

**Comment by madewild** [comment]
Yes @raphdom did some changes to the DEV>TEST workflow but it seems that it did not go well this night... the backend is broken in test.
In the meantime you can test on https://kohesio.development.ec.europa.eu/ or https://kohesio.acceptance.ec.europa.eu/

**Comment by Djang0** [comment]
<img width="1167" alt="image" src="https://user-images.githubusercontent.com/841633/154628043-c89718e0-f1f8-4c5f-9e69-8e559f3a0428.png">
development Environment.
<img width="1091" alt="image" src="https://user-images.githubusercontent.com/841633/154628139-39a636df-3176-4cfc-80cd-9aeb47d16e76.png">
acceptance environment...

are data cached at browser level (I don't really trust what I am writing here, but it is what it mimics)... somehow?  @madewild 

I can test test env later on if needed ....
